libs/blocks/section-metadata/section-metadata.css

```diff
@@ -68,29 +68,55 @@
   text-align: center;
 }
 
+.section.center[class*='-up'] {
+  justify-items: center;
+}
+
 .section.divider {
   border-bottom: 1px solid #d8d8d8;
 }
 
-.section.two-up .grid,
-.section.three-up .grid,
-.section.four-up .grid {
+.section.two-up,
+.section.three-up,
+.section.four-up,
+.section.five-up {
   display: grid;
   grid-template-columns: repeat(auto-fit, minmax(276px, 1fr));
+  gap: var(--spacing-m);
```

Comment:
I know @auniverseaway said simplified but I don't see why we wouldn't include the gap options. This is something Rikio already requested

Reply:
@meganthecoder @auniverseaway in my initial stab at this I created support for all the different consonant sizes for gap and column-gap/row-gap overrides. I can add all of them back in, or a subset of them. Any preferences?

Reply:
I think just adding the gap spacing tokens options will suffice. The column-gap and row-gap are nice to have, but might be more then we need for now.

Reply:
Okay I added back the section gap class tokens, we can add support for row or column if the use case arrises.